File: Searching_Algorithms/linear_search.py
def linear_search(lst,n,x):
    if x in lst:
        return lst.index(x)
    return -1



if __name__ == "__main__":
    print("Enter the total number of elements")
    n = int(input())
    lst = []
    print("Enter the elements separated by new line")
    for i in range(n):
        lst.append(int(input()))
    x = int(input("Enter the element to search: "))
    print(linear_search(lst,n,x))

File: Searching_Algorithms/linear_search.cpp
#include <iostream>

using namespace std;

int search(int arr[],int n, int x){
    for(int i =0;i<n;i++){
        if(arr[i]==x){
            return i;
        }
    }
    return -1;
}

int main(){
    int n, arr[50],i,x;
    cout << "Enter the size of the array: "<<endl;
    cin >> n;
    cout << "Enter the elements separated by new line" << endl;
    for(i=0;i<n;i++){
        cin >> arr[i];
    }
    cout << "Enter the element to search"<<endl;
    cin >> x;
    cout << "The element is available at index " << search(arr,n,x);
}
